contracts/exchange-wrappers/ZeroExExchangeWrapper.sol

```diff
@@ -0,0 +1,50 @@
+/*
+    Copyright 2021 dYdX Trading Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity ^0.8.0;
+
+
+import { I_ExchangeWrapper } from "../external/I_ExchangeWrapper.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ZeroExExchangeWrapper is I_ExchangeWrapper  {
+    // ============ Public Functions ============
+
+    /**
+     * Exchange some amount of takerToken for makerToken. Just for tests
+     *
+     * @param  tradeOriginator      Address of the initiator of the trade (however, this value
+     *                              cannot always be trusted as it is set at the discretion of the
+     *                              msg.sender)
+     * @param  receiver             Address to set allowance on once the trade has completed
+     * @param  makerToken           Address of makerToken, the token to receive
+     * @param  takerToken           Address of takerToken, the token to pay
+     * @param  inputTokenAmount     Amount of makerToken
+     * @param  orderData            Arbitrary bytes data for any information to pass to the exchange
+     * @return                      The amount of makerToken received
+     */
+    function exchange(
+        address tradeOriginator,
+        address receiver,
+        ERC20 makerToken,
+        ERC20 takerToken,
+        uint256 inputTokenAmount,
+        bytes calldata orderData
+    )
+        external override
+        returns (uint256)
+    {
+        // TODO implement a call to the actual ERC20 Transformation
+        return inputTokenAmount / uint(2);
+    }
+}
```

Comment:
Where are you getting this function signature from?

Can you link to what docs you're looking at? I was browsing 0x's docs and I found [this](https://protocol.0x.org/en/latest/advanced/erc20_transformations.html) for atomic ERC20 swaps, is this what you were looking at?

Reply:
I'm not getting it from anywhere - I was going to do that in a subsequent PR